### PR TITLE
Worker manager: split Azure scanner process

### LINKED ISSUE #5342: Create worker manager diagrams: sequence of calls

### PR DESCRIPTION
**Pull request by lotas**
* splitting worker scanner: 
       * one scanner to handle everything except azure
       * second one scans only workers of azure provider
* updating state and flow diagrams of worker manager (fixes #5342)

Github Bug: #5342, #4987

### CODE REVIEW

File: services/worker-manager/README.md [hunk 1]
```diff
@@ -136,6 +136,25 @@ end
   updateExpires -.-> cleanupProviders[Cleanup providers]
 ```
 
+#### Sequence of calls
```
Comment (matt-boris) [inline services/worker-manager/README.md:139]: https://github.com/taskcluster/taskcluster/blob/e8fde4f09e0f6a4f246556fa6d4a03abdf735f68/services/worker-manager/README.md#sequence-of-calls

**Review by matt-boris — APPROVED** [review]
Nice @lotas!! Just left a small question. Looks great though!

File: infrastructure/k8s/values.yaml [hunk 2]
```diff
@@ -195,6 +195,10 @@ worker_manager:
       replicas: 1
       cpu: 200m
       memory: 200Mi
+    workerscanner_azure:
+      replicas: 1
+      cpu: 50m
+      memory: 100Mi
```
[the remark this answers is not in the document]
Comment (lotas) [reply infrastructure/k8s/values.yaml:201]: Those were autogenerated and I guess just the minimum defaults in config. 
Ah, nicely spotted! I guess it makes sense to align both scanners to have similar cpu/mem requests :+1:

https://github.com/taskcluster/taskcluster/blob/5a25a717299e9eae61d7ed0935ceb028b8319a26/infrastructure/tooling/src/generate/generators/k8s.js#L82-L86

[the remark this answers is not in the document]
Comment (lotas) [reply infrastructure/k8s/values.yaml]: Will update those

**Review by matt-boris — APPROVED** [review]

**Pull request merged**